.github/scripts/trymerge.py

```diff
@@ -996,8 +998,23 @@ def find_matching_merge_rule(pr: GitHubPR,
     """Returns merge rule matching to this pr or raises an exception"""
     changed_files = pr.get_changed_files()
     approved_by = set(pr.get_approved_by())
-    rules = read_merge_rules(repo, pr.org, pr.project)
-    reject_reason = f"PR {pr.pr_num} does not match merge rules"
+
+    issue_link = (f"https://github.com/{pr.org}/{pr.project}/issues/new"
+                  f"?labels=module%3A%20ci&template=bug-report.yml")
```

Comment:
Hmm, can we move it a separate function (say `get_new_issue_link`), because I believe we construct something similar in other locations as well